### DISCUSSION TITLE
Old school self-taught programmer and New to GitHub

### DISCUSSION THREAD

**Discussion by philipabala** [community/community#102813] [New to GitHub]
Hello everyone,

I hope I'm posting in the right place, and if not, my apologies. As the headline suggests, I'm an old-school programmer who initially learned the ropes through books and on-the-job experience back in 1992. Coding fascinated me from the start, providing practical solutions for real-world problems and significantly boosting work efficiency. In 1998, I ventured into web development when the internet opened up to the public in the UAE. My first tool was MS FrontPage, followed by learning Classic ASP, which I've been using consistently—until my recent job role, where I encountered additional technologies like HTML, CSS, and JavaScript. All three were learned on the job, leaving me with a bit of a knowledge gap. I feel like I've been living under a rock, and now, as I catch up, I'm astonished by the transformations in the programming landscape. It's a bit overwhelming, and I find myself in a position where I need to double-time to truly catch up.

Thank you for your understanding and any guidance you can offer as I navigate this learning curve.

Best regards,
Philip

**Comment by queenofcorgis** [comment]
Welcome to GitHub and the GitHub Community @philipabala - thanks for sharing your journey with us 🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist. 

We also have a [Programming Help category](https://github.com/orgs/community/discussions/categories/programming-help) which is a great space for those HTML, CSS, and JavaScript conversations. 

You've got some great experience and perspective under your belt and I look forward to seeing you around the community 💟 . 

Best of luck!
